### DISCUSSION TITLE
Trackrecording only recording when osmand is anyway active

### DISCUSSION THREAD

**Discussion by bradfield123** [osmandapp/OsmAnd#19978] [Ideas]
Hello,

I use osmand for bike trips, and saving battery is important for me. I just have a track displayed and don't use navigation.
Track recording would be nice, but I don't activate it since I don't want to spend battery capacity due to osmand automatically recording points (every x minutes).

Since I anyway quite often check with osmand if I am still on the track, I was wondering if an option could be added to osmand track recording to only record a point when osmand was manually activated. So no automatic track recording. What do you think ?

**Comment by DmitryAlexei** [comment]
In your use case, GPS should be anyway active in the background...
You can try to set the General logging interval to 5 min and see if it is ok for you (accessing the GPS sensor will be less frequent, and battery consumption will be reduced accordingly).
https://osmand.net/docs/user/plugins/trip-recording#recording-settings

**Reply by bradfield123** [reply]
I can give it a try, but it would only be a workaround. And setting to 5 minutes is probably a bit too short, since this workaround would then only work, if I look at the screen at least every 5 minutes, which in my case is not always the case.

Better would be to set the time interval to very long and have an option to also record points whenever GPS sensor is active (OSMAND shown on screen)
